### PR TITLE
Fix to Baianus description

### PR DESCRIPTION
**Pull request by LazerLit**
As reported by [DoomKorath on Discord](https://discord.com/channels/251118043411775489/536900466655887360/1221444227796434964), the description of the planet Baianus erroneously refers to 'Titan' when it most likely should be referring to 'Triton'.

### CODE REVIEW

**Comment by Zitchas** [comment]
As the author of said description, no, I definitely intended Titan. 

I was focused much more on the nature of the planet itself than the atmosphere, as well as its proximity to the radiation belts around Saturn.

**Comment by LazerLit** [comment]
> As the author of said description, no, I definitely intended Titan.
> 
> I was focused much more on the nature of the planet itself than the atmosphere, as well as its proximity to the radiation belts around Saturn.

Ah, okay. I (and presumably Doom Korath and a few others) focused on the 'frozen water world' and 'tantalizing traces of chemicals that suggest that something more might exist below the surface' and immediately jumped to undersea oceans. Is it worth an edit to avoid confusion in the future? Perhaps...
```suggestion
	description `The surface of this frozen water world is little more than a shattered wasteland of ice and the occasional rock outcropping, but the unbreathably thin atmosphere contains countless tantalizing traces of chemicals that suggest that something more might exist below the surface.`
```
```suggestion
	description `	By size and composition Baianus is almost a twin to Titan back in Sol, and is similarly bathed in constant radiation. However, Baianus is regularly wracked by surges of both heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
```

**Comment by xX-Dillinger-Xx** [comment]
This is funny. People are worried about whether a reference in the Baianus description should be Titan or Triton. It's nice to be accurate but there are a lot bigger accuracy issues then this to worry about. The most noticeable is planet vs ship scale. Another, issue is that there a lot of planet and moon images that don't even match their description.

**Comment by Saugia** [comment]
Adjusting a description to better fit the intention is easier to work with than large-scale changes to the game's identity.

For reference, accuracy with scale is not an issue in Endless Sky as realistic scale comparison is not something we are pursing for any part of the game. We work with the best we can regarding images and landscapes, but those to don't have to be 100% accurate either. ES isn't a simulator or accurate representation of Sci-Fi, and has many old-school arcade-like elements to it.

In about a week we plan on releasing our Vision Document, which should help define what exactly we're looking at for the game's direction. Regardless, there's nothing wrong with taking a look at descriptions or other cases like this on the side while we work on larger changes to the game at the same time (currently balance changes such as the mass PR).

**Comment by xX-Dillinger-Xx** [comment]
Maybe, I didn't use the best examples but it's just what popped in my head. I wasn't saying that it should or shouldn't be changed. I just found it funny that, of all the accuracy issues, someone even noticed such a trivial thing and even took the time to comment on it.  I know that may sound a little hypocritical, coming from me, but that's my issue to work on. I've reached the age were I just call it as I see it.

**Review by OcelotWalrus — APPROVED** [review]

**Comment by Zitchas** [comment]
Given that we haven't ever actually visited Titan (or Triton), I'd just as soon leave the description alone. 

The idea is to provoke mystery and thought, and imply enough that the player can imagine it. As such, I don't find any small discrepancies to be any sort of problem, but rather a plus. We're not there, after all. We have to imagine it. The play of similarities vs differences can work more on the human mind than pure similarities.

**Comment by xX-Dillinger-Xx** [comment]
@Zitchas 
> Given that we haven't ever actually visited Titan (or Triton), I'd just as soon leave the description alone.

I feel the decision should be yours to make, but ....................well I won't finish that sentence. 

>  The play of similarities vs differences can work more on the human mind than pure similarities.

I agree, but lets be honest I doubt many have even read the description. That's why I found it so funny that anybody even took the time to care.

**Comment by Amazinite** [comment]
The way this description is structured, the first paragraph should be listing similarities to Titan ("This frozen water world is reminiscent of Titan back in Sol: [similarities]"), while the second paragraph is listing differences ("Unlike Titan, however, [differences]"). A thin atmosphere shouldn't be listed in the "similarities" paragraph, as Titan's atmosphere is actually thicker than Earth's, hence the confusion. Therefore, either we just shouldn't mention the thickness of the atmosphere, or the fact that it's thin should be listed in the second paragraph.

Even baring that, though, Triton does sound like a better fit for a comparison given what's described. Triton has an icy terrain and an incredibly thin atmosphere. Titan, as mentioned, has a thicker atmosphere, and I'd say it's most striking terrain feature is its methane lakes, not its icy terrain, so I wouldn't exactly consider a planet lacking in methane lakes (which Baianus likely can't have due to its thin atmosphere) to be reminiscent of Titan.

**Comment by roadrunner56** [comment]
Considering the comments made by Zitchas and Derpy above, I'm closing this PR as it seems to fix a problem that wasn't really there in the first place.

**Comment by roadrunner56** [comment]
Sorry, I misread the PR. Ignore me.

File: data/map planets.txt [hunk 1]
```diff
@@ -286,8 +286,8 @@ planet Babiali
...
-	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
+	description `This frozen water world is reminiscent of Triton back in Sol: the surface is little more than a shattered wasteland of ice and the occasional rock outcropping, but the unbreathably thin atmosphere contains countless tantalizing traces of chemicals that suggest that something more might exist below the surface.`
+	description `	Unlike Triton, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
```
Comment (MasterOfGrey) [inline data/map planets.txt:290]: @Zitchas suggested restructuring of the information for clarity:

```suggestion
	description `From a distance this large, icy moon is superficially reminiscent of Titan back in Sol: the surface is little more than a shattered wasteland of ice and the occasional rock outcropping, it is similarly exposed to it's giant parent's radiation belts, and scans indicate countless tantalizing traces of chemicals that suggest that something more might exist below the surface.`
	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras swirl through it's unbreathably thin atmosphere, accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
```

Comment (MasterOfGrey) [reply data/map planets.txt:290]: From a distance in stellar terms, it'd be about the same mass, orbiting in a relatively similar range.
Those are the first two things you can tell from a distance with the least amount of detection equipment.

Sure the *next* thing you'd notice is the difference in atmosphere which is very different, but I don't think the point is invalid.

... Europa might be a better match up close though, albeit one that is substantially smaller than Titan.

Comment (MasterOfGrey) [reply data/map planets.txt:290]: Personally, mass is the main thing I knew about Titan, but the principle of this suggestion was just to restructure Zitchas's original intent along the lines Derpy proposed for aiding comprehension.

If you really want to debate for a different stellar body entirely, you can have that debate with someone else.

Comment (Zitchas) [reply data/map planets.txt:290]: I'm good with MasterOfGrey's proposed changes here.

Comment (warp-core) [reply data/map planets.txt:290]: > From a distance in stellar terms, it'd be about the same mass, orbiting in a relatively similar range.
Those are the first two things you can tell from a distance with the least amount of detection equipment.

I don't think this explanation lines up with the proposed description very well.
If size and orbital distance are the similarities the opening line of the description alludes to, it seems odd to elaborate on that statement of superficial reminiscence by describing the surface features. Any similarities in the surface would only be visible past the differences in the atmospheres, and, at the point of considering the atmospheric differences, it's difficult to say calling "reminiscent of Titan," is a reasonable description.
The description of the surface features also ignores perhaps the most unique property of Titan's surface, it's hydrocarbon lakes, and focuses on properties that are common to a number of other moons in our system.

Comment (Zitchas) [reply data/map planets.txt:290]: I have no strong feelings either way. If you really want to change it, go for it. I don't think it improves anything, but I don't think it makes anything worse, either. Unnecessary, but neutral.

**Comment by Zitchas** [comment]
I have no strong feelings either way. If you really want to change it, go for it. I don't think it improves anything, but I don't think it makes anything worse, either. Unnecessary, but neutral.

File: data/map planets.txt (hunk 1 above)
Comment (MasterOfGrey) [reply data/map planets.txt:290]: I have updated the above suggestion with deference to Warp's notes.

Comment (Zitchas) [reply data/map planets.txt:290]: Trust looks good.

**Review by MasterOfGrey — APPROVED** [review]

File: data/map planets.txt [hunk 2]
```diff
@@ -286,8 +286,8 @@ planet Babiali
...
-	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
+	description `From a distance this large, icy moon is superficially reminiscent of Titan back in Sol: it is similarly exposed to it's giant parent's radiation belts, and scans indicate countless tantalizing traces of chemicals that suggest there may be something more to it than it appears.`
+	description `	Unlike Titan, however, the surface of  Baianus is little more than a shattered wasteland of ice and rock that is regularly wracked by both waves of heat and energy. A stay of any length on the surface is likely to see waves of auroras swirl through it's unbreathably thin atmosphere, accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
```
Comment (MasterOfGrey) [inline data/map planets.txt:290]: Fixing the check error.
```suggestion
	description `	Unlike Titan, however, the surface of Baianus is little more than a shattered wasteland of ice and rock that is regularly wracked by both waves of heat and energy. A stay of any length on the surface is likely to see waves of auroras swirl through it's unbreathably thin atmosphere, accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
```

File: data/map planets.txt [hunk 3]
```diff
@@ -286,8 +286,8 @@ planet Babiali
 planet Baianus
 	attributes frozen uninhabited
 	landscape land/snow20
-	description `This frozen water world is reminiscent of Titan back in Sol: the surface is little more than a shattered wasteland of ice and the occasional rock outcropping, but the unbreathably thin atmosphere contains countless tantalizing traces of chemicals that suggest that something more might exist below the surface.`
-	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
+	description `From a distance this large, icy moon is superficially reminiscent of Titan back in Sol: it is similarly exposed to it's giant parent's radiation belts, and scans indicate countless tantalizing traces of chemicals that suggest there may be something more to it than it appears.`
```
Comment (TheMarksman-ES) [inline data/map planets.txt:289]:
```suggestion
	description `From a distance, this large, icy moon is superficially reminiscent of Titan back in Sol: it is similarly exposed to its giant parent's radiation belts, and scans indicate countless tantalizing traces of chemicals that suggest there may be something more to the moon than it appears.`
```

**Comment by Quantumshark** [comment]
> I was focused much more on the nature of the planet itself than the atmosphere, as well as its proximity to the radiation belts around Saturn.

Everything I can find suggests that Titan isn't in or near any of Saturn's radiation belts, so as an analogy for proximity to radiation belts it doesn't really fit as a comparison either. Enceladus is, I think, the best fit for both your original description and the features you've ascribed to Titan - it's just within Saturn's radiation belt, its surface is 'little more than a shattered wasteland of ice', and although its atmosphere is more 'nonexistent' than 'unbreathably thin' (that point would more closely describe Triton) there are certainly 'tantalizing traces of chemicals' there.

I'd also like to make the point that even from a distance, Baianus doesn't resemble Titan. Titan is a fuzzy blob of indistinct orange haze.
image
Baianus is a cratered rocky moon that uses an image of Callisto.
image
So, that first line, `From a distance, this large, icy moon is superficially reminiscent of Titan back in Sol`, isn't really accurate.

File: data/map planets.txt [hunk 4]
```diff
@@ -286,8 +286,8 @@ planet Babiali
...
-	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
+	description `From a distance, this large, icy moon is superficially reminiscent of Titan back in Sol: it is similarly exposed to its giant parent's radiation belts, and scans indicate countless tantalizing traces of chemicals that suggest there may be something more to the moon than it appears.`
+	description `	Unlike Titan, however, the surface of Baianus is little more than a shattered wasteland of ice and rock that is regularly wracked by both waves of heat and energy. A stay of any length on the surface is likely to see waves of auroras swirl through it's unbreathably thin atmosphere, accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
```
Comment (Quantumshark) [inline data/map planets.txt:290]:
```suggestion
	description `From a distance, this large, icy moon is superficially reminiscent of Enceladus back in Sol: it is similarly exposed to its giant parent's radiation belts, and scans indicate countless tantalizing traces of chemicals that suggest there may be something more to the moon than it appears.`
	description `	Unlike Enceladus, however, the surface of Baianus is regularly wracked by both waves of heat and energy. A stay of any length on the surface is likely to see waves of auroras swirl through it's unbreathably thin atmosphere, accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
```

**Comment by tibetiroka** [comment]
Closing as orphaned. Feel free to reopen this PR once the review comments are addressed.

**Pull request closed without merging**